### PR TITLE
AT_04.09.07 | Each field has a capital letter of the Latin alphabet

### PR DESCRIPTION
**Pull request by LIliia149**
https://trello.com/c/6XgmqCsC/719-at040907-create-booking-page-departure-date-each-field-has-a-capital-letter-of-the-latin-alphabet

### CODE REVIEW

File: cypress/e2e/testUiAndFunction/createBookingPage/US_04.09_CalendarAvailableDaysWeekUI.cy.js [hunk 1]
```diff
@@ -37,33 +37,47 @@ describe('US_04.09 | Calendar available days week UI', () => {
...
+	it('AT_04.09.07 | Each field has a capital letter of the Latin alphabet', function () {
+
+		createBookingPage.getCalendarDays().then($els => {
+
+			for (let i = 0; i < 7; i++) {
+				const win = $els[i].ownerDocument.defaultView
+				const before = win.getComputedStyle($els[i], 'before')
+				const contentValue = before.getPropertyValue('content')
+				
+				expect(contentValue).to.deep.equal(contentValue.toUpperCase())
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/createBookingPage/US_04.09_CalendarAvailableDaysWeekUI.cy.js:79]: The assert is not very correct. You get the result and compare it with him. Better create an array with the values ​​of the days of the week in the fixture and then check against it.

**Comment by LIliia149** [comment]
Please look at the changes - there are quite a lot because I used 'Format document' and it added some spaces.

File: cypress/e2e/testUiAndFunction/createBookingPage/US_04.09_CalendarAvailableDaysWeekUI.cy.js (hunk 1 above)
Comment (LIliia149) [reply cypress/e2e/testUiAndFunction/createBookingPage/US_04.09_CalendarAvailableDaysWeekUI.cy.js:79]: But the point of the case to verify that each field has a capital letters - check if there are special days of the week is the point of the next TC

Comment (Rom-T) [reply cypress/e2e/testUiAndFunction/createBookingPage/US_04.09_CalendarAvailableDaysWeekUI.cy.js:79]: Sorry, didn't look.

**Review by Rom-T — DISMISSED** [review]

File: cypress/e2e/testUiAndFunction/createBookingPage/US_04.09_CalendarAvailableDaysWeekUI.cy.js (hunk 1 above)
Comment (LIliia149) [reply cypress/e2e/testUiAndFunction/createBookingPage/US_04.09_CalendarAvailableDaysWeekUI.cy.js:79]: @Rom-T Thank you, Roman!

**Review by YurySelivanov — APPROVED** [review]

**Review by Rom-T — APPROVED** [review]

**Review by EkaterinaRomankova — APPROVED** [review]

**Pull request merged**